packages/gatsby-source-shopify/src/nodes.js

```diff
@@ -38,7 +42,7 @@ const downloadImageAndCreateFileNode = async (
   }
 
   const fileNode = await createRemoteFileNode({
-    url,
```

Comment:
we can just skip over this code if `downloadFiles` is set to false

Reply:
awesome, yeah not being hugely familiar with the code base I wasn't sure if I could just skip over it or if I would break something else where if a image wasn't provided, hence why I supplied a placeholder image in its place. which the cache would result in only one image being downloaded rather than all of the images.

thank you so much for pointing this out.